src/nvim/terminal.c

```diff
@@ -217,10 +214,14 @@ Terminal *terminal_open(TerminalOptions opts)
   vterm_set_utf8(rv->vt, 1);
   // Setup state
   VTermState *state = vterm_obtain_state(rv->vt);
+  // Only relevant in terminal true color mode
+  if (true_color) {
+    vterm_state_set_bold_highbright(state, true);
```

Comment:
but wouldn't `g:terminal_bold_highbright` only be used in true color mode anyways?

I mean, in non true color mode there is no reason to have highbright on because the user terminal does the color shifting anyways.

But perhaps you want me to move it down to the below the `(!true_color)` check and have an additional check for `g:terminal_bold_highbright`?